main.py

```diff
@@ -139,9 +155,10 @@ def BoardInit(args):
 
                 # Send messages from executed logic
                 BoardShim.log_message(LogLevels.LEVEL_DEBUG.value, "Sending")
-                send_pairs = osc_reporter.send(data_dict)
-                for param_path, param_value in send_pairs:
-                    BoardShim.log_message(LogLevels.LEVEL_DEBUG.value, "{}:\t{:.3f}".format(param_path, param_value))
+                for reporter in reporters:
+                    send_pairs = reporter.send(data_dict)
+                    for param_path, param_value in send_pairs:
+                        BoardShim.log_message(LogLevels.LEVEL_DEBUG.value, "{}:\t{:.3f}".format(param_path, param_value))

```

Comment:
This loop could functionally be removed if the above suggestion is applied, as the reporters list only ever seems to have one present in it anyways